web/src/utils/factory-management/factory.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { newFactory } from '@/utils/factory-management/factory'
+import { countIncompleteTasks, newFactory } from '@/utils/factory-management/factory'
```

Comment:
Typically, I advise developers to use positive variable names, as this is essentially a double negative, can cause conditional logic confusion later. Recommend `countActiveTasks` instead of `countIncompleteTasks` in this context.